extensions/contact-forms/shortcodes/contact-form/class-fw-shortcode-contact-form.php

```diff
@@ -13,6 +15,15 @@ public function _init()
 			'fw_option_type_builder:page-builder:register_items',
 			array($this, '_action_register_builder_item_types')
 		);
+
+		add_filter( 'fw_ext:shortcodes:collect_shortcodes_data', array(
+			$this, 'add_contact_form_data_to_filter'
+		) );
+	}
+
+	public function add_contact_form_data_to_filter( $structure ) {
```

Comment:
Please prefix methods with `_filter_` or `_action_`.
Because in my IDE (PhpStorm), when I write:

```php
/** @var FW_Shortcode_Contact_Form $contact_form */
$contact_form = fw_ext('shortcodes')->get_shortcode('contact-form');
$contact_form-> // here IDE shows autocomplete
```

and it will be confusing if a method has a regular name and it's public, but actually it is meant to be private used only in action or filter.

Also would be nice to set the `@internal` phpDoc

```php
/**
 * @internal
 */
public function _filter_add_contact_form_data_to_filter($structure) {
    // ...
}
```